File: PHP PowerPoint Examples/SlidesApiDemo/getSlide.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "GetSlide.pptx";
$slideIndex = 2;
$password = null; 
$folder = "Powerpoint"; 
$storage = null; 
$result = $apiInstance->getSlide($name, $slideIndex, $password, $folder, $storage);

File: Python PowerPoint Examples/SlidesApiDemo/getSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "GetSlide.pptx"
password = ""
storage = ""
slide_index = 2
inputFolder = "input"
result =api.get_slide(fileName, slide_index=slide_index, password=password, storage=storage, folder=inputFolder)